charts/dns/values.yaml

```diff
@@ -30,9 +31,10 @@ externalDNS:
       memory: 256Mi
 
 pdns:
-  repository: ghcr.io/txqueuelen/pdns-stateless
+  registry: ghcr.io
+  repository: txqueuelen/pdns-stateless
+  tag: v4.4
   pullPolicy: Always
```

Comment:
As above

Reply:
As above, fixed in [#24e5b71](https://github.com/txqueuelen/pdns-stateless/pull/8/commits/24e5b712363b9bdce5c1dedd93033f0cb430371f)